app/_includes/landing_pages/plugin_card.md

```diff
@@ -0,0 +1,7 @@
+{% assign plugin_name = include.config.plugin %}
```

Comment:
@lmilan we already have a component for plugin cards, see [example](https://github.com/Kong/developer.konghq.com/blob/546-ai-gateway-landing-page/app/_landing_pages/gateway/rate-limiting.yaml#L140-L142), [implementation](https://github.com/Kong/developer.konghq.com/blob/main/app/_includes/landing_pages/plugin.md)

Reply:
can you use that one instead?